src/pages/stacks-blockchain/stacking.md

```diff
@@ -77,3 +77,34 @@ Stacking is a built-in capability of PoX and is realized through a set of action
 ## Stacking contract
 
 Check out the [Stacking contract reference](/references/stacking-contract) to see available methods and error codes.
+
+## Bitcoin address
+
+The Stacking contract needs a special format for the bitcoin address (the reward address). This is required in order to ensure that miners will be able to correctly construct the bitcoin transaction containing the reward address.
+
+!> An invalid reward address could cause miners to be unable to mine a block
```

Comment:
I'm not sure if this warning makes sense here-- an invalid reward address wouldn't be accepted by the PoX contract at all. The reason that this validation is performed is to prevent invalid addresses from making a block impossible to mine.